iothub_client/inc/iothub_device_client.h

```diff
@@ -349,6 +348,88 @@ extern "C"
     */
     MOCKABLE_FUNCTION(, IOTHUB_CLIENT_RESULT, IoTHubDeviceClient_UploadMultipleBlocksToBlobAsync, IOTHUB_DEVICE_CLIENT_HANDLE, iotHubClientHandle, const char*, destinationFileName, IOTHUB_CLIENT_FILE_UPLOAD_GET_DATA_CALLBACK_EX, getDataCallbackEx, void*, context);
 
+     /**
+    * @brief    This API creates a context for a new blob upload to Azure Storage.
```

Comment:
Please use the same formula as other APIs associated with the blob object. i.e. something like "Creates a context for...".

Reply:
I don't get it, it is following the format of the other function documentation. Could you elaborate on what you mean?

Reply:
I think this is outdated. Please re-comment if needed.